mmtrack/models/reid/linear_reid_head.py

```diff
@@ -33,16 +39,39 @@ def __init__(self,
                  act_cfg=None,
                  num_classes=None,
                  loss=dict(type='CrossEntropyLoss', loss_weight=1.0),
+                 loss_pairwise=dict(
```

Comment:
The default losses should be all None, nor the users will be always using the loss if they ignore the argument